### DISCUSSION TITLE
[Question]:  Tag cyclists as persons with Frigate+ ?

### DISCUSSION THREAD

**Discussion by jchretien** [blakeblackshear/frigate#13138] [Ask A Question]
### What is your question:

Without Frigate+ bicycles can be detected, but now that I'm using Frigate+ bicycles are detected as persons.

What I've been doing was to keep tagging them as person.

You guys said in the future it will be be possible to have more labels and even custom labels, so I'm wondering if those past images submitted will confuse the models if they are not re-labelled for the new labels.

If I want to detect the bicycles, cyclists, runners, walkers, motorcycles, motoriders, will I need to re-evaluate all the past images?

I'm wondering for the present if I should avoid submitting images who may cause headache later.

**Comment by NickM-27** [comment]
you should not mislabel things, when more labels are added there will be a new workflow for going back through images that were previously verified.

**Reply by jchretien** [reply]
What do you mean by mislabelling?

**Reply by NickM-27** [reply]
Labeling something that it isn't, like labeling a raccoon as a cat

**Reply by jchretien** [reply]
Did I say I labelled racoons as cats? Please explain why you say that.

**Reply by NickM-27** [reply]
it was just an example. Another example would be putting a `person` box around the entire bicycle. It should only be around the person
